### ISSUE TITLE
Can't send Up arrow or navigate history?

### ISSUE THREAD

**Issue by jorxster** [ionelmc/python-remote-pdb#30]
Hi Ionel,
Thanks so much for making pyremotepdb, I use it as an alternate to GUI debuggers as I usually need to debug other processes over the network. 

I was just wondering if it's a known issue (or limitation of telnet/nc) that one cannot use the Up arrow keyboard shortcut to reload the last run command. Indeed none of the arrow keys work, simple keyboard shortcuts don't work in my case. (all on Linux, kubuntu 18, etc).
I'm not intimately familiar with how stdin works so perhaps this is a limitation that can't be overcome? Would be a shame, I'd love to have remotepdb work just like pdb.

cheers,

**Comment by jorxster** [comment]
For example, pressing arrow keys into telnet once connected gives me :
`^[[A^[[A^[[B^[[B^[[D^[[A^[[C^[^]`

**Comment by ionelmc** [comment]
Yes, that's because there's no pty on the remote side, nor should it - remove-pdb aims to be as simple and reliable as possible. There are alternatives you can use if you want a proper terminal.

**Comment by jorxster** [comment]
Ah, thank you, what alternatives are there?
Will have to look into pty.

**Comment by ionelmc** [comment]
Have a look at:
- https://github.com/ionelmc/python-manhole#similar-projects
- https://github.com/ionelmc/python-remote-pdb#similar-projects

Or search pypi: https://pypi.org/search/?q=pdb+telnet&o=
